### PR TITLE
button routes to dummy data objects

### PR DESCRIPTION
**Pull request by Hamed-TAYEBI**
dummy set entries must be accessible from the main page

### CODE REVIEW

File: components/ActivityCards.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import Image from "next/image";
 import Link from "next/link";
 import TagList from "./TagList/TagList";
+import { useRouter } from "next/router";
```
Comment (mbosselmann) [inline components/ActivityCards.js:4]: I think you do not use `useRouter` here or? :) Then the import can be removed.

File: components/ActivityCards.js [hunk 2]
```diff
@@ -45,6 +46,12 @@ export default function ActivityCards({ activities, editTag }) {
                   </Link>
                 )}
               </li>
+              <li>
+                {" "}
+                <button>
```
Comment (mbosselmann) [inline components/ActivityCards.js:51]: And you use a button element or a link element. The button element is not necessary. You can style your link element like a button if it the appearance of a button that you want to have.

**Review by mbosselmann — COMMENTED** [review]
Hi Hamed,
nice code! 💪 But I left some remarks for you to consider before moving further. :octocat:

File: pages/_app.js [hunk 3]
```diff
@@ -7,9 +7,7 @@ export default function App({ Component, pageProps }) {
   function addActivity(newActivity) {
     setActivities([{ ...newActivity, id: crypto.randomUUID() }, ...activities]);
   }
-  function editTag(activityId, tagId) {
-    console.log("I am app", activityId, "my id is", tagId);
-  }
+  function editTag(activityId, tagId) {}
```
[the remark this answers is not in the document]
Comment (Hamed-TAYEBI) [reply pages/_app.js:10]: appreciated pointing out the styling issues in detail...as to this function, it is used in my upcoming user stories.

**Pull request merged**